src/test/java/e2e/AddPhoneTest.java

```diff
@@ -0,0 +1,69 @@
+package e2e;
+
+import e2e.pages.*;
+import org.junit.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+
+
+public class AddPhoneTest extends BaseTest {
+    LoginHelper loginHelper;
+    ContactsPage contactsPage;
+    AddContactDialog addContactDialog;
+    ContactInfoPage contactInfoPage;
+    PhonesPage phonesPage;
+    AddNewPhoneNumberDialog addNewPhoneNumberDialog;
```

Comment:
It will be enough to call AddPhoneNumberDialog to make it more compact

Reply:
Fixed